window.go

```diff
@@ -37,3 +47,11 @@ func (w *Window) ListPanes() ([]Pane, error) {
 func (w *Window) AddPane(pane Pane) {
 	w.Panes = append(w.Panes, pane)
 }
+
+// Sets the window layout. The possible value can be one of the contants:
+// LayoutEvenVertical, LayoutEvenHorizontal, LayoutMainVertical,  LayoutMainHorizontal, LayoutTiled
+// See: https://www.man7.org/linux/man-pages/man1/tmux.1.html#WINDOWS_AND_PANES
+
```

Comment:
Please remove a new line to make a docstring comment from it

Reply:
fixed